portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Dhruvi!</h1>");
```

Comment:
I definitely should have mentioned this earlier but next time you'll want to wait for the first PR to be submitted before merging the second one. Otherwise these changes end up getting pulled into unrelated PRs.